experiments/templates/experiments/overview.html

```diff
@@ -55,10 +98,60 @@ <h1>
         <p>
             {% trans 'experiments:home:info_text' %}
         </p>
+        <form id="list_query_form">
+            <div class="list-controls mb-2">
+                <div style="flex: 1">
+                    <label>{% translate 'experiments:home:sort' %}
+                        <select name="sort">
+                            <option value="date_created" {%if request.GET.sort == 'date_created' %}selected{%endif%}>
+                                {% translate 'experiments:home:sort:date:desc' %}
+                            </option>
+                            <option value="-date_created" {%if request.GET.sort == '-date_created' %}selected{%endif%}>
+                                {% translate 'experiments:home:sort:date:asc' %}
+                            </option>
+                        </select>
+                    </label>
+                </div>
+                <label>
+                    {% translate 'experiments:home:search' %}
+                    <input name="search" value="{{ request.GET.search }}"></input>
+                </label>
+                {% if request.GET.search %}
+                <button class="clear ms-2">{% translate 'experiments:home:search:clear' %}</button>
+                {% endif %}
+            </div>
+        </form>
 
-        <div id="list"></div>
+        <div id="list">
 
-        <br/>
+            {% for experiment in object_list %}
+                <div>
+                    {% include 'experiments/fragment.html' %}
```

Comment:
perhaps rename to experiments/(experiment_)list_item.html? As a fragment is perhaps a bit of a vague name.